src/App.tsx

```diff
@@ -3,13 +3,6 @@ import { Global, css } from "@emotion/core";
 import GQLWrapper from "gql/GQLWrapper";
 import { BrowserRouter as Router } from "react-router-dom";
 import { ErrorBoundary } from "components/ErrorBoundary";
-import {
-  getGQLUrl,
-  getSchemaString,
-  isDevelopment,
-  isTest,
-  shouldEnableGQLMockServer,
-} from "utils/getEnvironmentVariables";
```

Comment:
can we remove any of these unused functions from `utils/getEnvironmentVariables` and anything related to `mockIntrospectSchema` and `mockCustomSchema` in our ReadMe?

Reply:
done